defender/secret-example.yml

```diff
@@ -0,0 +1,12 @@
+keys:
+  api: <API Key goes here>
+  secret: <API Secret goes here>
+
+secrets:
+  # Twitter Keys
+  # Consumer Keys from an elevated developer account
+  appKey: ""
```

Comment:
Should this be `apiKey` and `apiSecret`?

Reply:
Twitter and `TwitterApi` use the property `appKey` so `app` is appropriate here.